src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure/Commands/ASPNetBenchmarks/AspNetBenchmarksAnalyzeCommand.cs

```diff
@@ -64,10 +84,10 @@ public override int Execute([NotNull] CommandContext context, [NotNull] AspNetBe
                 }
             }
 
-            // Launch new process.
             Dictionary<string, string> benchmarkToComparisons = new();
             Dictionary<string, List<MetricResult>> metricResults = new();
 
+            // Compute the compare functionality if there is a single run.
```

Comment:
I don't understand this or the following added comment about a "single run" after the `return new()` above.